iina/PlayerCore.swift

```diff
@@ -135,6 +136,7 @@ class PlayerCore: NSObject {
         mpvController.setFlag(MPVOption.PlaybackControl.pause, true)
         setSpeed(1)
       }
+      statusPaused = !statusPaused
```

Comment:
Seems that `statusPaused` is an unused field. Please delete it directly, and use `playerCore.info.isPaused` instead.